### PR TITLE
[INLONG-5296][Manager] Add fields for DataNode and InlongCluster

### LINKED ISSUE #5296: [Improve][Manager] Add fields for DataProxy cluster info
### Description

Add fields for DataProxy cluster info.

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by healchow**
### Prepare a Pull Request
*(Change the title refer to the following example)*

- Fixes #5296

### Motivation

Add fields for DataNode and InlongCluster.

### Verifying this change

- [x] This change is already covered by existing tests, such as:
    DataNodeServiceTest

### CODE REVIEW

**Review by leosanqing — APPROVED** [review]

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/cluster/InlongClusterServiceImpl.java [hunk 1]
```diff
@@ -215,7 +215,7 @@ public Boolean updateTag(ClusterTagRequest request, String operator) {
                     String updateTags = Joiner.on(",").join(tagSet);
                     entity.setClusterTags(updateTags);
                     entity.setModifier(operator);
-                    int rowCount = clusterMapper.updateByIdSelective(entity);
+                    int rowCount = clusterMapper.updateById(entity);
                     if (rowCount != InlongConstants.AFFECTED_ONE_ROW) {
```
Comment (haibo-duan) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/cluster/InlongClusterServiceImpl.java:219]: Personally，local variable 'rowCount' is redundant.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/cluster/InlongClusterServiceImpl.java [hunk 2]
```diff
@@ -405,7 +405,7 @@ public Boolean bindTag(BindTagRequest request, String operator) {
                 InlongClusterEntity updateEntity = clusterMapper.selectById(id);
                 updateEntity.setClusterTags(updateTags);
                 updateEntity.setModifier(operator);
-                int rowCount = clusterMapper.updateByIdSelective(updateEntity);
+                int rowCount = clusterMapper.updateById(updateEntity);
                 if (rowCount != InlongConstants.AFFECTED_ONE_ROW) {
```
Comment (haibo-duan) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/cluster/InlongClusterServiceImpl.java:409]: same as above

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/cluster/InlongClusterServiceImpl.java [hunk 3]
```diff
@@ -793,7 +793,7 @@ private void removeClusterTag(InlongClusterEntity entity, String clusterTag, Str
         String updateTags = Joiner.on(",").join(tagSet);
         entity.setClusterTags(updateTags);
         entity.setModifier(operator);
-        int rowCount = clusterMapper.updateByIdSelective(entity);
+        int rowCount = clusterMapper.updateById(entity);
         if (rowCount != InlongConstants.AFFECTED_ONE_ROW) {
```
Comment (haibo-duan) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/cluster/InlongClusterServiceImpl.java:797]: same as above

**Review by ciscozhou — APPROVED** [review]

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/cluster/InlongClusterServiceImpl.java (hunk 1 above)
Comment (healchow) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/cluster/InlongClusterServiceImpl.java:219]: done.

**Review by haibo-duan — APPROVED** [review]

**Review by woofyzhao — APPROVED** [review]

**Review by kipshi — APPROVED** [review]

**Pull request merged**